apps/cli/src/__tests__/__examples__/action-backtick.ts

```diff
@@ -1,13 +1,13 @@
-import { createMachine } from "xstate";
+import { createMachine } from 'xstate';
```

Comment:
If you ever reach a point where you don't have much work in flight, we should update and merge #202 so we don't have this formatting crap littering so many of your PRs 😆

Reply:
Rebased and fixed :P